### DISCUSSION TITLE
Musical Lyric

### DISCUSSION THREAD

**Discussion by DeafNet** [chidiwilliams/buzz#1019] [Ideas]
Is there such thing that ai can recognize the vocal that sung and turn into subtitle ? Just curious?

**Comment by raivisdejus** [comment]
Currently no, there is no feature to extract vocals from the audio. You would need to do that manually with some tool like https://github.com/Anjok07/ultimatevocalremovergui

**Comment by raivisdejus** [comment]
@DeafNet The very latest development version adds option to extract speech from audio before it is transcribed. See this how to get the latest development version https://chidiwilliams.github.io/buzz/docs/faq#9-where-can-i-get-latest-development-version and then enable this option when you add new audio file to transcribe
